languages.toml

```diff
@@ -2369,3 +2368,16 @@ language-server = { command = "cs", args = ["launch", "com.disneystreaming.smith
 [[grammar]]
 name = "smithy"
 source = { git = "https://github.com/indoorvivants/tree-sitter-smithy", rev = "cf8c7eb9faf7c7049839585eac19c94af231e6a0" }
+
+[[language]]
+name = "puppet"
+scope = "source.puppet"
+injection-regex = "puppet"
+file-types = ["puppet"]
+roots = []
+indent = { tab_width = 2, unit = "  " }
+comment-token = "#"
```

Comment:
file-types = ["pu"]
indent = { tab-width = 2, unit = " " }

Reply:
I would actually argue that it is Pascal (or, specifically, FreePascal's need to be different) which is in conflict with Puppet here. Historically and practically, `.pas` is the recognized Pascal extension. `.pp` is just a vendor-specific variant which doesn't really have a rational reason to exist in the first place and therefore it should be the other way around.